### DISCUSSION TITLE
Question about the output figures

### DISCUSSION THREAD

**Discussion by hu-lm** [deepmodeling/deepmd-kit#1557] [Q&A] [answered]
Hi,
I am writing this discussion because I have a question when looking at the output figure,
![waterexamplefig](https://user-images.githubusercontent.com/98113070/157506618-59729dcb-6998-40ae-8e15-d87aaf00cbfd.jpg)
Here there are six lines. I understand that rmse_e_trn/val and rmse_f_trn/val stand for loss for energy and force, but what are rmse_val/trn represent for (the upper two lines)? Thank you so much!

**Comment by njzjz** [comment, marked as answer]
It's the root of the total loss. The equation of the total loss can be found in [this paper](https://doi.org/10.48550/arXiv.2203.00393) (Eq. 3).
